webpack.config.js

```diff
@@ -229,6 +229,7 @@ const setup = {
 			// Most dependencies will be bundled for the AMP setup screen for compatibility across WP versions.
 			requestToHandle: ( handle ) => {
 				switch ( handle ) {
+					case 'lodash':
```

Comment:
Note: if these changes don't fix the issue in #5147 then we may instead want to emulate the approach in Site kit here: https://github.com/google/site-kit-wp/pull/1153/files
This would mean going back to bundling lodash into our JS.